Project.toml

```diff
@@ -1,6 +1,6 @@
 name = "Oceananigans"
 uuid = "9e8cae18-63c1-5223-a75c-80ca9d6e9a09"
-version = "0.62.2"
+version = "0.62.3"
```

Comment:
This is a new feature. Shouldn't we bump the version to 0.63.0 instead? Or are there bug fixes?